cli/src/main.rs

```diff
@@ -1,6 +1,9 @@
 #[macro_use]
 extern crate clap;
+#[macro_use]
```

Comment:
this means you import the macros defined in this crate?

Reply:
I read about macros and it seems like its no longer needed if we use rust 2018, will fix